### ISSUE TITLE
Smoothing Bot -> Human interaction.

### ISSUE THREAD

**Issue by wprzyb** [TeMPOraL/alice#10]
" Alice_M: TeMPOraL: dalsza część obliczeń wygląda na sporą, wysłać ci ją na query, czy wypisać tutaj ? "
- if the output is bigger than it should be, Alice should limit her output to f.ex only first line (because it can have solution in it), then ask if you want the rest of output on the channel or via query - if query, she can throw whole query there, doesn't matter. But if i choose channel, she should also then limit her output. It's the matter of deciding how much i want and making noice minimal.
- Another idea kind of contradicts with previous one, but the main goal is to make alice similar to human by customizing time of responding to $(ammount of letters) \* $(average geek typing speed). It can be used with the smaller queries, which also should be compressed (if there is small ammount of characters per line) from f.ex 3 to one, and maybe optically split by using ^C stuff.

tl;dr:
- Alice handling:
1. small outputs: 
   1. compress
   2. optically nice, aesthetic
2. big outputs:
   1. safely assume that the first line is good, print it, and ask if somebody wants to get the rest of the output on query or here, on channel. 
   2. first_line = next_line
   3. goto (iii).
